expenses-manager-backend/src/main/java/com/endava/expensesmanager/controller/ExpenseController.java

```diff
@@ -0,0 +1,57 @@
+package com.endava.expensesmanager.controller;
+
+import com.endava.expensesmanager.entity.Category;
+import com.endava.expensesmanager.entity.Currency;
+import com.endava.expensesmanager.entity.Expense;
+import com.endava.expensesmanager.entity.Users;
+import com.endava.expensesmanager.service.ExpenseService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/expense")
+public class ExpenseController {
+    private final ExpenseService expenseService;
+
+    public ExpenseController(ExpenseService expenseService) {
+        this.expenseService = expenseService;
+    }
+
+    @PostMapping
+    public ResponseEntity<Expense> createExpense(@RequestParam LocalDateTime date, @RequestParam Category category,
```

Comment:
All the expense fields should come as a request body - check `@RequestBody` annotation. The object we receive should be a new DTO object.